conf_linux_public.py

```diff
@@ -18,53 +18,30 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-"""
-Media SDK open source product configuration
-
-See the API documentation at the end of this configuration
-
-Root dir Layout:
-root_dir
-    repos
-        repo1
-        repo2
-    build
-    install
-    logs
-        clean
-            _all.log
-            name1.log
-            name2.log
-        extract
-            _all.log
-            repo1.log
-            repo2.log
-        build
-            _all.log
-            name1.log
-            name2.log
-        install
-            _all.log
-            name1.log
-"""
 
 PRODUCT_REPOS = [
     {'name': 'MediaSDK'},
     #{'name': 'flow_test'},
 ]
 
+COMMANDS = [
```

Comment:
Not good name. Should be like
ENABLE_DEV_TOOL_SET='source /opt/rh/devtoolset-6/enable'
Also please don't use feature list of commands. It should be removed from the build scripts.